<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>325</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Ubuntu Condensed</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>440</width>
     <height>325</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>1000</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>671</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>181</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>COM Port</string>
      </property>
      <widget class="QComboBox" name="FlowControlBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>170</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lP">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>140</y>
         <width>32</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Parity:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lDB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>80</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Data bits:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="ParityBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="StopBitsBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>110</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnConnect">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>230</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Conect</string>
       </property>
      </widget>
      <widget class="QComboBox" name="DataBitsBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="lFC">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>62</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Flow control:</string>
       </property>
      </widget>
      <widget class="QLabel" name="lBR">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>51</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>BaudRate:</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnDisconect">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>230</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Disconect</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnSearch">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>72</width>
         <height>24</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnDefault">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>200</y>
         <width>151</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Default Value</string>
       </property>
      </widget>
      <widget class="QComboBox" name="PortNameBox">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>21</y>
         <width>69</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="BaudRateBox">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>51</y>
         <width>69</width>
         <height>20</height>
        </rect>
       </property>
       <property name="currentIndex">
        <number>-1</number>
       </property>
      </widget>
      <widget class="QLabel" name="lSB">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>110</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop bits:</string>
       </property>
      </widget>
      <widget class="QToolButton" name="toolButton">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>160</y>
         <width>32</width>
         <height>19</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>271</height>
       </rect>
      </property>
      <property name="title">
       <string>Log</string>
      </property>
      <widget class="QTextBrowser" name="textBrowserLog">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>171</width>
         <height>211</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnClearLog">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clear</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxTimer">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>0</y>
        <width>271</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Timer</string>
      </property>
      <widget class="QLabel" name="labelMin">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>10</y>
         <width>32</width>
         <height>16</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="text">
        <string>min</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxSeconds">
       <property name="geometry">
        <rect>
         <x>83</x>
         <y>30</y>
         <width>63</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Titillium Web SemiBold</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="spinBoxMinutes">
       <property name="geometry">
        <rect>
         <x>12</x>
         <y>30</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Titillium Web SemiBold</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="frame">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="readOnly">
        <bool>false</bool>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelSec">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>32</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>sec</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnStartTimer">
       <property name="geometry">
        <rect>
         <x>155</x>
         <y>20</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnStopTimer">
       <property name="geometry">
        <rect>
         <x>155</x>
         <y>48</y>
         <width>72</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBoxHeaterAndAir">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>80</y>
        <width>271</width>
        <height>191</height>
       </rect>
      </property>
      <property name="title">
       <string>Heater and Air</string>
      </property>
      <widget class="QSpinBox" name="spinBoxSettingHeaterTemper">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>87</y>
         <width>71</width>
         <height>41</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Titillium Web SemiBold</family>
         <pointsize>30</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="accelerated">
        <bool>false</bool>
       </property>
       <property name="correctionMode">
        <enum>QAbstractSpinBox::CorrectToPreviousValue</enum>
       </property>
       <property name="showGroupSeparator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>59</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelEstablTemperNotMatter">
       <property name="geometry">
        <rect>
         <x>15</x>
         <y>67</y>
         <width>91</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Establised temper.</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelCelsium2">
       <property name="geometry">
        <rect>
         <x>87</x>
         <y>83</y>
         <width>51</width>
         <height>42</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>°C</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnSetTemperature">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>70</y>
         <width>65</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>SET</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonAutoMode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>51</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Auto</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButtonManualMode">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>20</y>
         <width>61</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Manual</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButtonWorkMode">
       <property name="geometry">
        <rect>
         <x>126</x>
         <y>18</y>
         <width>53</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelBlow">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>32</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Blow</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnBlowMode">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>150</y>
         <width>53</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>ON</string>
       </property>
      </widget>
      <widget class="QPushButton" name="BtnReadTemperature">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>100</y>
         <width>65</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>17</pointsize>
        </font>
       </property>
       <property name="text">
        <string>READ</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>170</x>
         <y>153</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Sound at end</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <attribute name="title">
      <string>Graph</string>
     </attribute>
     <widget class="QLabel" name="labelNowTemperNotMatter">
      <property name="geometry">
       <rect>
        <x>553</x>
        <y>4</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Now temperature</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCelsium1">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>20</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>°C</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNowTemperature">
      <property name="geometry">
       <rect>
        <x>558</x>
        <y>20</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelEstablTemperNotMatter_2">
      <property name="geometry">
       <rect>
        <x>553</x>
        <y>110</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Establised temper.</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelCelsium1_2">
      <property name="geometry">
       <rect>
        <x>610</x>
        <y>126</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>°C</string>
      </property>
     </widget>
     <widget class="QLabel" name="labelNowTemperature_2">
      <property name="geometry">
       <rect>
        <x>558</x>
        <y>126</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="labelTimeLeft">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>200</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Time left: </string>
      </property>
     </widget>
     <widget class="QLabel" name="labelForTimeLeft">
      <property name="geometry">
       <rect>
        <x>560</x>
        <y>209</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>30</pointsize>
       </font>
      </property>
      <property name="text">
       <string>00:00</string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="customPlot" native="true">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>251</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDebug">
     <property name="accessibleName">
      <string/>
     </property>
     <attribute name="title">
      <string>Debug</string>
     </attribute>
     <widget class="QCheckBox" name="checkBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>acceptanse for state</string>
      </property>
     </widget>
     <widget class="QPushButton" name="findAndConnectButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Find &amp;&amp; Connect</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
